vizro-ai/docs/pages/user-guides/run-vizro-ai.md

```diff
@@ -81,36 +81,97 @@ There are two ways to integrate Vizro-AI into an application, directly and by ac
+
+!!! example "Application integration via chart code"
+
+    === "app.py"
+        ```py
+        import vizro.plotly.express as px
+        from vizro_ai import VizroAI
+
+        vizro_ai = VizroAI()
+
+        df = px.data.gapminder()
+        code_string = vizro_ai._get_chart_code(df, "describe life expectancy per continent over time")
+        ```
+    === "code_string"
+        [![ResultCode]][ResultCode]
 
-        [ResultCode]: ../../assets/user_guides/code_string_app_integration.png
+    [ResultCode]: ../../assets/user_guides/code_string_app_integration.png
 
-    The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
+The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
 
-    To use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
+To use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
 
 ### How to use `max_debug_retry` parameter in plot function
 - Default Value: 3
-- Type: int
+- Type: `int`
 - Brief: By default, the `max_debug_retry` is set to 3, the function will try to debug errors up to three times.
 If the errors are not resolved after the maximum number of retries, the function will stop further debugging retries.
 For example, if you would like adjust to 5 retries, you can set `max_debug_retry = 5` in the plot function:
 
 ```py
 vizro_ai.plot(df = df, user_input = "your user input", max_debug_retry= 5)
 ```
+
+### How to use `return_elements` parameter in plot function
+- Default Value: False
+- Type: `bool`
+- Brief: By default, the `return_elements` is set to `False`, and the `VizroAI.plot()` returns a `plotly.graph_objects` object. If `return_elements` is set to `True` `VizroAI.plot()` returns the `PlotOutputs` data class.
+The `PlotOutputs` data class is designed to encapsulate all possible outputs generated by the `VizroAI.plot()` method. This class ensures that all outputs are organized and accessible.
+
+Attributes of `PlotOutputs`:
+
+- **`code`**: A string representing of the Python code that manipulates the data and creates the visualization.
+- **`figure`**: An instance of a Plotly `go.Figure` object, representing the visual plot generated by `VizroAI.plot()`.
```

Comment:
Hm, technically it is a CapturedCallable.. Maybe we can say that and link to the Vizro docs, and explain that this is so it immediately runs in a dashboard. Otherwise it just behaves like a plotly `go.Figure`